omemo/litesessionstore.py

```diff
@@ -86,3 +89,17 @@ def deleteAllSessions(self, recipientId):
         q = "DELETE FROM sessions WHERE recipient_id = ?"
         self.dbConn.cursor().execute(q, (recipientId, ))
         self.dbConn.commit()
+
+    def setActiveState(self, deviceList, jid):
+        c = self.dbConn.cursor()
+
+        q = "UPDATE sessions SET active = {} " \
+            "WHERE recipient_id = '{}' AND device_id IN ({})" \
+            .format(TRUE, jid, ', '.join(['?'] * len(deviceList)))
```

Comment:
you mean for the JID?
cause devicelist is escaped

originally i wanted to input also the jid with a ? but it was not working
that doesnt work:
`c.execute(q, (jid, deviceList))

devicelist is a list of unknown length, if i write it like that, execute treats devicelist like it holds only one device.
maybe there is way i dont know